cmds/chmod/chmod.go

```diff
@@ -13,32 +13,89 @@ package main
 
```

Comment:
This looks like a good improvement, but could we get the chmod changes in a separate PR?

Reply:
Got a question here. I know the GNU programmers have lots of longoptions that are identical to a short option, but I'm not sure I see a point to replicating that practice. Part of the reason they did this, I suspect, is that the long option is a kind of documentation and the GNU longopts structure has no place for a doc string, which the Go flag struct does. So a long opt, called recursive, is somewhat helpful given that you have no other help text. 
Given this, I think we should drop the -recursive flag, since -R does the same thing, and -h will tell you about the -R option anyway. 
Sound OK? We've had some discussions about this at Google in previous years and I think that's our general rule.
Thanks. If you really want that -recursive switch, I guess we'll do it, but I wanted to at least ask.

I guess reference is OK, if a bit wordy, because there is no equivalent short option (or is there?).